lib/Parser.js

```diff
@@ -324,14 +324,16 @@ Parser.prototype = {
         debug('Pagination...');
         return this._paginator
             .paginate()
-            .then(function (paginatoin) {
-                if (paginatoin.done) {
+            .then(function (pagination) {
```

Comment:
Yes, but generally this is about spelling replacement "paginatoin" -> "pagination"